README.md

```diff
@@ -39,6 +42,9 @@ This would be possible in the mobility domain by making sure all mobility apps i
 * Added TICKET_INFO tag group for ticket number
 * Added ref_id in status call
 * Added parent_stop_id for stops
+* Added timestamp for time in items.
+* Added separate fulfillment as TICKET for each ticket quantity with base fulfillment as type TRIP
```

Comment:
dynamic TICKET fulfillment associated with each authorizationand, customer details. Support 1 ticket for multiple quantity and also multiple tickets for multiple tickets